refinery/data_set_manager/utils.py

```diff
@@ -17,12 +17,15 @@
 from django.db.models import Q
 from django.utils.http import urlquote, urlunquote
 
+from constants import REFINERY_SOLR_DOC_LIMIT
```

Comment:
import constants and move it into the "local" import section below since this is not a 3rd party module